torch/testing/_internal/opinfo/definitions/linalg.py

```diff
@@ -1657,6 +1665,11 @@ def make_input():
                 device_type="mps",
                 dtypes=[torch.float32],
             ),
+            DecorateInfo(
+                unittest.skip("cpu vs gpu is sometimes different"),
+                "TestCommon",
+                "test_compare_cpu",
+            ),
```

Comment:
Yeah, `lstsq` is a tricky one. In particular, it's known to be non-deterministic on CPU. See https://github.com/pytorch/pytorch/issues/71222. Can you mention this issue in the skip?